packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Component/Input.tsx

```diff
@@ -30,6 +30,7 @@ const ComponentInput = ({
   attribute,
   disabled,
   labelAction,
+  renderLayout = ComponentLayout,
```

Comment:
I think you should be setting the default at the same level it is being used and it can be undefined until then

Reply:
sure I can change it, I just wonder why you prefer that?

Reply:
My previous comment is maybe no good. I had an error yesterday when running this branch and I believe it was related to this being undefined. So I guess I mean just make sure it's defined where needed :)